lms/djangoapps/django_comment_client/tests/test_utils.py

```diff
@@ -118,7 +120,7 @@ class CoursewareContextTestCase(ModuleStoreTestCase):
     comment client service integration
     """
     def setUp(self):
-        super(CoursewareContextTestCase, self).setUp(create_user=True)
+        super(CoursewareContextTestCase, self).setUp()
```

Comment:
`ModuleStoreTestCase` is what sets the baseline default for `CREATE_USER`, and it sets it to `True`. (Don't ask my why the MSTestCase is the one creating users... but that ship has sailed).

So, you only need to override it if you don't want the users created.